final_task/rss_reader/classes/rss_item.py

```diff
@@ -0,0 +1,39 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class RssItem:
+    '''
+    Represents a single news from RSS channel
+    '''
+    title: str
+    published: str
+    link: str
+    media: str
+
+    def format_data(self, data):
+        '''
+        Method replaces ASCII codes by their char representations 
+        '''
+        data = data.replace('&#60;', '<')
+        data = data.replace('&#x3c;', '<')
+        data = data.replace('&#x3C;', '<')
+        data = data.replace('&#62;', '>')
+        data = data.replace('&#x3e;', '>')
+        data = data.replace('&#x3E;', '>')
+        data = data.replace('&#38;', '&')
+        data = data.replace('&#x26;', '&')
+        data = data.replace('&#34;', '"')
+        data = data.replace('&#x22;', '"')
+        data = data.replace('&#39;', '\'')
+        data = data.replace('&#x27;', '\'')
```

Comment:
It is better to find a more universal solution is some libraries. There may be other symbols which have encoding problems.

Reply:
Done